web/packages/teleterm/src/sharedProcess/ptyHost/ptyEventsStreamHandler.ts

```diff
@@ -112,8 +112,13 @@ export class PtyEventsStreamHandler {
         })
       );
     });
-    this.ptyProcess.start(event.columns, event.rows);
-    this.logger.info(`stream has started`);
+    // PtyProcess.prototype.start always returns a fulfilled promise. If an error is caught during
+    // start, it's reported through PtyProcess.prototype.onStartError. Similarly, the information
+    // about a successful start is also conveyed through an emitted event rather than the method
+    // returning with no error. Hence why we can ignore what this promise returns.
+    void this.ptyProcess.start(event.columns, event.rows).then(() => {
```

Comment:
I'm using the syntax recommended by no-floating-promises to explicitly mark a promise that we ignore. https://typescript-eslint.io/rules/no-floating-promises/#ignorevoid